nixos/modules/services/hardware/xow.nix

```diff
@@ -0,0 +1,31 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.hardware.xow;
+in {
+  options.services.hardware.xow = {
+    enable = lib.mkOption {
```

Comment:
`lib.mkEnableOption` is convenient for this.

Reply:
Didn't know about that, thanks!